### ISSUE TITLE
Recovered Stateflow Charts

### ISSUE THREAD

**Issue by geez0x1** [geez0x1/CompliantJointToolbox#15]
The `Recovered Stateflow Charts` issue keeps popping up, despite the `deleteStateflowChart.m` code. 

Errors that can be seen while closing models relate to the same issue:
```
Warning: Error evaluating 'ModelCloseFcn' callback of SubSystem block (mask) 'my_model/JointModel'. 
 
Warning: Cannot open 'my_model' while it is being closed. 
 
Warning: Error evaluating 'ModelCloseFcn' callback of SubSystem block (mask) 'my_model/JointModel1'. 
 
Warning: Cannot open 'my_model' while it is being closed. 
```

**Comment by joernmalzahn** [comment]
Does not look simple. Having a hard time to reproduce it. It might be a Simulink Version Issue. Will take some time, but I do agree, we have to fix this.

**Comment by geez0x1** [comment]
@joernmalzahn Any progress on this?

**Comment by joernmalzahn** [comment]
No.

**Comment by geez0x1** [comment]
This remains an issue for me even in R2017a.

**Comment by joernmalzahn** [comment]
I have not seen any recovered stateflow char block in quite a while. @geez0x1 Is this still happening to you?

The error messages that you quote are gone with #58

**Comment by geez0x1** [comment]
Yes this still happens to me. A reliable way to reproduce it seems to be opening a model (that does not have orphaned StateFlow charts), simulating it, saving it, and closing it. Re-opening will then show an orphaned chart.

Removing the chart, saving, closing, and repeating a second time, appears to fairly reliably remove the orphaned chart.
